package.json

```diff
@@ -76,19 +82,22 @@
     "@types/react": "^17.0.40",
     "@types/react-dom": "^17.0.13",
     "babel-loader": "^8.2.3",
+    "dts-cli": "^1.6.0",
     "framer-motion": "^6",
     "gh-pages": "^3.2.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-is": "^17.0.2",
     "semantic-release": "^19.0.2",
     "size-limit": "^7.0.8",
-    "dts-cli": "^1.6.0",
     "tslib": "^2.3.1",
     "typescript": "^4.6.2"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/perxhealth/react-ui.git"
+  },
+  "dependencies": {
```

Comment:
we've migrated to the up-to-date tsdx fork since this PR began, so the issue which necessitated us installing prettier ourselves should no longer be a worry. let's revert `d14037`, should also remove that pesky `~/.npmrc` issue